packages/codegen/src/generators/react-typescript/app/template/src/core/i18n/messages/en.json

```
@@ -11,9 +11,11 @@
   "common.back": "Back",
   "common.previous": "Previous",
   "common.requestFailed": "Request failed",
+  "common.validationError": "Invalid form field value",
```

There can be multiple invalid fields, so it's better to use more generic:

```suggestion
  "common.validationError": "Validation error",
```